browser/ui/views/frame/vertical_tab_strip_root_view.cc

```diff
@@ -0,0 +1,73 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
Oh, my, I'm really sorry that I misunderstood about this file. When I read the [diff between the commits](https://github.com/brave/brave-core/pull/19785/files/7f5af90f798810dea41c5efe5f92fc8f9181484d..0e2bb9a6fd5e1a92f2dbed38665e530fb5007fa1#diff-a21ae1b6c91082e4a86117318cb8cb96d08056a57b5dacdfece631358be81463), I thought this file were not a new one, but an already existing one. So I said "this should be 2022" but I was wrong. If this is new file, this should be 2023 as you initially submit. I apologize for making you work again.